pkg/disttask/framework/scheduler/interface.go

```diff
@@ -26,7 +26,7 @@ import (
 // TaskManager defines the interface to access task table.
 type TaskManager interface {
 	// GetTopUnfinishedTasks returns unfinished tasks, limited by MaxConcurrentTask*2,
-	// to make sure lower rank tasks can be scheduled if resource is enough.
+	// to make sure low ranking tasks can be scheduled if resource is enough.
```

Comment:
I prefer using "high priority" in the description, "ranking" is unclear to readers.

Reply:
`priority` is only part of how we order tasks, initially all task have the same priority, so i use rank to clearify it